repair comment router fails build:repair on missing pr-close-coverage-proof module

## Summary

The ClawSweeper repair/comment router is failing before it can route acknowledged commands. This is currently blocking Telecrawl automerge commands on:

- openclaw/telecrawl#4
- openclaw/telecrawl#6
- openclaw/telecrawl#7

Those PRs have green CI and ClawSweeper-ready verdicts, but GitHub still reports `autoMergeRequest: null` because the router never gets past build setup.

## Evidence

Recent failed router runs:

- https://github.com/openclaw/clawsweeper/actions/runs/26698708512
- https://github.com/openclaw/clawsweeper/actions/runs/26698708398
- https://github.com/openclaw/clawsweeper/actions/runs/26698708309

Failure from run 26698708512:

```
src/repair/apply-result.ts(43,8): error TS2307: Cannot find module '../pr-close-coverage-proof.js' or its corresponding type declarations.
```

Step: `Run ./.github/actions/setup-pnpm`, command `pnpm run build:repair`.

## Expected

The router should compile and process acknowledged `@clawsweeper automerge` comments, or update the command comment with an actionable failure instead of leaving it stuck at "Command router queued".

## Notes

No Telecrawl code repair is indicated by the current ClawSweeper reviews; this appears to be a ClawSweeper router build issue.

--- a/.github/workflows/repair-comment-router.yml
+++ b/.github/workflows/repair-comment-router.yml
@@ -85,9 +85,14 @@ jobs:
             .github
             config
             jobs
+            prompts/pr-close-coverage-proof.md
             results
+            schema/clawsweeper-pr-close-coverage-proof.schema.json
             scripts/hydrate-state.ts
+            src/clawsweeper-text.ts
+            src/codex-env.ts
             src/github-retry.ts
+            src/pr-close-coverage-proof.ts
             src/repair
             src/repository-profiles.ts
             package.json

--- a/.github/workflows/spam-comment-intake.yml
+++ b/.github/workflows/spam-comment-intake.yml
@@ -53,7 +53,12 @@ jobs:
           sparse-checkout: |
             .github/actions
             config/automation-limits.json
+            prompts/pr-close-coverage-proof.md
+            schema/clawsweeper-pr-close-coverage-proof.schema.json
+            src/clawsweeper-text.ts
+            src/codex-env.ts
             src/github-retry.ts
+            src/pr-close-coverage-proof.ts
             src/repair
             src/repository-profiles.ts
             package.json

--- a/.github/workflows/spam-scanner.yml
+++ b/.github/workflows/spam-scanner.yml
@@ -73,9 +73,14 @@ jobs:
           sparse-checkout: |
             .github
             config
+            prompts/pr-close-coverage-proof.md
             results
+            schema/clawsweeper-pr-close-coverage-proof.schema.json
             scripts/hydrate-state.ts
+            src/clawsweeper-text.ts
+            src/codex-env.ts
             src/github-retry.ts
+            src/pr-close-coverage-proof.ts
             src/repair
             src/repository-profiles.ts
             package.json

--- a/test/repair/workflow-sparse-checkout.test.ts
+++ b/test/repair/workflow-sparse-checkout.test.ts
@@ -1,0 +1,57 @@
+import assert from "node:assert/strict";
+import fs from "node:fs";
+import path from "node:path";
+import test from "node:test";
+
+const REPAIR_PROOF_RUNTIME_PATHS = [
+  "prompts/pr-close-coverage-proof.md",
+  "schema/clawsweeper-pr-close-coverage-proof.schema.json",
+  "src/clawsweeper-text.ts",
+  "src/codex-env.ts",
+  "src/pr-close-coverage-proof.ts",
+] as const;
+
+const SPARSE_REPAIR_BUILD_WORKFLOWS = [
+  ".github/workflows/repair-comment-router.yml",
+  ".github/workflows/spam-comment-intake.yml",
+  ".github/workflows/spam-scanner.yml",
+] as const;
+
+test("sparse repair build workflows include PR close proof runtime files", () => {
+  for (const workflowPath of SPARSE_REPAIR_BUILD_WORKFLOWS) {
+    const workflow = fs.readFileSync(path.join(process.cwd(), workflowPath), "utf8");
+    assert.match(workflow, /build-script: build:repair/);
+
+    const entries = sparseCheckoutEntries(workflow);
+    for (const requiredPath of REPAIR_PROOF_RUNTIME_PATHS) {
+      assert.ok(entries.has(requiredPath), `${workflowPath} missing ${requiredPath}`);
+    }
+  }
+});
+
+function sparseCheckoutEntries(workflow: string): Set<string> {
+  const entries = new Set<string>();
+  const lines = workflow.split(/\r?\n/);
+
+  for (let index = 0; index < lines.length; index += 1) {
+    const line = lines[index] ?? "";
+    if (!/^\s+sparse-checkout:\s*\|/.test(line)) continue;
+
+    const blockIndent = leadingSpaces(line);
+    for (index += 1; index < lines.length; index += 1) {
+      const entryLine = lines[index] ?? "";
+      if (!entryLine.trim()) continue;
+      if (leadingSpaces(entryLine) <= blockIndent) {
+        index -= 1;
+        break;
+      }
+      entries.add(entryLine.trim());
+    }
+  }
+
+  return entries;
+}
+
+function leadingSpaces(value: string): number {
+  return value.length - value.trimStart().length;
+}

